exports/gensymbol

```diff
@@ -3053,34 +3053,34 @@
     ## @(MATGEN_OBJ) from `lapack-3.4.1/lapacke/src/Makefile`
     ## Not exported: requires LAPACKE_TESTING to be set and depends on libtmg
     ##               (see `lapack-3.4.1/TESTING/MATGEN`).
-    LAPACKE_clatms,
```

Comment:
These `MATGEN` objects are not built by default in `lapack` right?

Reply:
The Makefiles do build them by default (while the netlib cmakefiles do have a separate BUILD_TESTING option as far as I recall)